_posts/2020-05-20-gsoc-students.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: post
+title: Google Summer of Code Students 2020
+---
+
+We are happy to anounce that MDAnalysis is hosting three [GSoC][gsoc]
+students this year -- @hmacdope, @cbouy, and @yuxuanzhuang. This is
+the first year that MDAnalysis has been accepted as its own
+[organization with GSoC][mda-gsoc] and we are grateful to Google for granting us
+three student slots so that we can have _three exciting GSoC
+projects_.
+
+## Hugo MacDermott-Opeskin: Trajectory New Generation: the trajectory format for the future of simulation
+
+<img
+src="https://pbs.twimg.com/profile_images/1140166815919665152/D-x2Tvae_400x400.jpg"
+title="Hugo MacDermott-Opeskin" alt="Hugo MacDermott-Opeskin"
+style="float: left; width: 110px; height: 110px; border-radius: 20px; border: 15px solid white" />
+
+Trajectory storage has always proved problematic for the molecular
+simulation community, as large volumes of data can be generated
+quickly. Traditional trajectory formats suffer from poor portability,
+large file sizes and limited ability to include metadata relevant to
+simulation. The Trajectory New Generation (TNG) format developed by
+the [GROMACS][] team represents the first trajectory format with small
+file sizes, metadata storage, archive integrity verification and
+user/software signatures. The primary goal of [this
+project](https://summerofcode.withgoogle.com/projects/#5116604104310784)
+is for @hmacdope to refactor the existing TNG code into C++ to provide
+clarity and usability for GROMACS, other simulation packages and
+analysis tools. Thin FORTRAN and Python layers are also desirable to
+encourage widespread adoption and are a secondary goal of the
+project. An efficient and transferable implementation of the TNG
+format will represent a major step forward for the computational
+molecular sciences community, enabling easy storage and replication of
+simulations.
+
+This project is a collaboration with the [GROMACS][] developer team
+with @acmnpv from GROMACS serving as a co-mentor.
+
+Hugo MacDermott-Opeskin is a PhD student in computational chemistry at
+the Australian National University. His work focuses on studying
+membrane biophysics through molecular dynamics simulations coupled
+with enhanced sampling techniques.  Hugo can be found on github as
+[@hmacdope][hmacdope-github] and on twitter as
```

Comment:
@IAlibay you don't need to explicitly link the GitHub – we have an extension that automatically expands the @-handle.

Reply:
Oops my apologies, will revert that (and those below)